app/src/main/java/com/mmm/parq/fragments/DriverHomeFragment.java

```diff
@@ -71,11 +69,9 @@
     static private final int COARSE_LOCATION_PERMISSION_REQUEST_CODE = 1;
     static private final int FINE_LOCATION_PERMISSION_REQUEST_CODE = 0;
     static private final int LINE_WIDTH = 20;
-    static private final int MAPS_REQUEST_CODE = 1;
     static private final int ZOOM_LEVEL = 16;
 
     private static final String TAG = DriverHomeFragment.class.getSimpleName();
-    private static String CLASS = "DriverHomeFragment";
 
     public enum State {
         FIND_SPOT, NAVIGATION, ARRIVE_SPOT, OCCUPY_SPOT, END_RESERVATION
```

Comment:
Just saw this, but yeah I think there sort of is (it's just off by one). 

What i'm thinking right now is: 
DriverFindSpotFragment in FIND_SPOT state calls /reserve -- > DriverAcceptFragment
DriverAcceptFragment in RESERVED state calls /accept --> DriverOccupyFragment
DriverOccupyFragment in ACCEPTED state calls /occupy --> DriverFinishFragment
DriverFinishFragment in OCCUPIED state calls /finish --> DriverReviewFragment
DriverReviewFragment in FINISHED state calls /review --> DriverFindSpotFragment

although now that I've typed that out I'm starting to think this might make more sense: 
DriverFindSpotFragment in FIND_SPOT state calls /reserve --> DriverReservedFragment
DriverReservedFragment in RESERVED state calls /accept --> DriverAcceptedFragment
DriverAcceptedFragment in ACCEPTED state calls /occupy --> DriverOccupiedFragment
DriverOccupiedFragment in OCCUPIED state calls /finish --> DriverFinishedFragment
DriverFinishedFragment in FINISHED state calls /review --> DriverFindSpotFragment

I dunno cause in the second one the naming matches the state more, but it doesn't match what the fragment is doing as much...